README.md

```diff
@@ -91,6 +91,10 @@ A curated list of awesome inclusive events/projects/initiatives for women in the
   - [Women Who Code](https://www.womenwhocode.com) - *Loc:* <img src="https://upload.wikimedia.org/wikipedia/en/a/a4/Flag_of_the_United_States.svg" alt="United States" width="30">
 
     Women Who Code (WWCode) is a U.S. based 501(c)(3) nonprofit dedicated to inspiring women to excel in technology careers. We connect amazing women with other like minded amazing women around the globe who unite under one simple notion – the world of technology is much better with women in it.
+    
+    - [Bay Area Girls](http://girlgeek.io/about/)
+    
+    Company sponsored dinners in the San Francisco Bay Area which invite women from various areas of technology to share their experiences, alongside prominent speakers from the female tech community, networking and potential recruiting by the sponsoring company. It has been hosted by 100+ companies and has several events per month.
```

Comment:
Maybe: Sponsored dinners in San Francisco where the women coming all over are invited to share their experience...

What do you think?